include/solvers/time_solver.h

```diff
@@ -225,6 +225,19 @@ class TimeSolver : public ReferenceCountedObject<TimeSolver>
    */
   void set_solution_history(const SolutionHistory & _solution_history);
 
+  /**
+   * A setter function for the replace_stored_entry boolen owned by
+   * the solution history object
+   * */
+  void set_solution_history_replace_stored_entry(bool val)
+  { solution_history->set_replace_last_stored(val); }
```

Comment:
This seems way too specific, doesn't it?  Especially since we now have a getter, so get_solution_history()->set_replace_last_stored() is possible?

Reply:
Makes sense. Will remove the redundant setter.